code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -133,6 +133,8 @@
 /obj/item/weapon/implant/dust
 	name = "duster implant"
 	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
 
```

Comment:
@Crazylemon64 Right now, Dust implants create an icon at the top left of the screen. As do many other types of implants. Unfortunately, prior to this PR they had no icon defined, so the icon they generated was blank. That was bad. This PR includes a bugfix which changes them to have a pile of bones icon. This visually represents what they do when activated. It is also consistent with the explosive implant, whose icon is an explosion.